torch/csrc/profiler/api.h

```diff
@@ -15,6 +15,7 @@ namespace impl {
 enum class C10_API_ENUM ActivityType {
   CPU = 0,
   CUDA, // CUDA kernels, runtime
+  HPU,  // HPU kernels
```

Comment:
I think you have to have exactly one space between the comma and comment. I'm not 100% sure; clang format is complaining but is tight lipped about why.